content/en/apps/guides/hosting/monitoring.md

```diff
@@ -79,3 +79,161 @@ Messaging Outgoing State Failed | `non-zero` | 0
+
+### Upgrading
+
+Before upgrading, you should back up both your current configuration settings and your Prometheus time-series data.
+
+#### Prometheus, Grafana and JSON Exporter
+
+To upgrade these dependencies, update the version numbers set in your `.env` file (or leave them set to `latest`).  Then run the following commands:
+
+```shell
+docker compose pull
+docker compose up -d
+```
+
+#### CHT Monitoring Config
+
+When you see a new version in the [GitHub repository](https://github.com/medic/cht-monitoring), first review the release notes and upgrade instructions. Then, run the following commands to deploy the new configuration:
+
+1. Note the version number to derive the branch to pull.  For example,  `1.1.0` would be `1.1.x`
+2. Run this command to update your instance and restart it. Be sure to replace `BRANCH_NAME` with the value from the first step:
+   ```shell
+   cd ~/cht-monitoring
+   git fetch
+   git checkout BRANCH_NAME
+   ```shell
+
+### Additional Configuration
+
+#### Prometheus Retention and Storage
+
+By default, historical monitoring data will be stored in Prometheus (`PROMETHEUS_DATA` directory) for 60 days (configurable by `PROMETHEUS_RETENTION_TIME`). A longer retention time can be configured to allow for longer-term analysis of the data.  However, this will increase the size of the Prometheus data volume.  See the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/storage/) for more information.
+
+Local storage is not suitable for storing large amounts of monitoring data. If you intend to store multiple years worth of metrics, you should consider integrating Prometheus with a [Remote Storage](https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage/).
+
+#### Alerts
+
+This configuration includes number of pre-provisioned alerts.  Specific alerting rules (and other contact points) can be set in the Grafana UI.  
+
+See the both Grafana [high level alert Documentation](https://grafana.com/docs/grafana/latest/alerting/) and  [provisioning alerts in the UI](https://grafana.com/docs/grafana/latest/alerting/set-up/provision-alerting-resources/file-provisioning/#provision-alert-rules) for more information on how to edit or remove these provisioned alerts.
+
+Additionally, you can configure where these alerts are sent.  Two likely options are Email and Slack.
+
+##### Email
+
+To support sending email alerts from Grafana, you must update the `smtp` section of your `grafana/grafana.ini` file with your SMTP server configuration.  Ten, in the web interface, add the desired recipient email addresses in the `grafana-default-email` contact point settings.
+
+##### Slack
+
+Slack alerts can be configured within the Grafana web GUI for the specific panel you would like to alert on.  Find the panel you want to alert on and edit it.  In the bottom half of the screen, choose the "Alert" tab. Then click "Create alert fule from this panel".
```

Comment:
```suggestion
Slack alerts can be configured within the Grafana web GUI for the specific rules you would like to alert on.
```

>  Find the panel you want to alert on and edit it.  In the bottom half of the screen, choose the "Alert" tab. Then click "Create alert fule from this panel".

This feels like odd information to include in the Slack section since it is simply general info on how to create an alert from a panel. They should be able to trigger Slack notifications for existing alerts (and email notifs for alerts created from panels...).

Reply:
Agreed!